contents/blog/startup-ops-toolkit.md

```diff
@@ -0,0 +1,93 @@
+
+<iframe src="https://giphy.com/embed/o2xbqBmHbQAbm" width="480" height="269" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/move-donald-trump-o2xbqBmHbQAbm">via GIPHY</a></p>
+
+### Accounting
+
+For your accounting, you basically want to do the bare minimum to be legally compliant and not sued by the IRS; and have financial reports to look at so you can nod and go ‘hm, yes we are still burning money at an alarming rate.’
+
+We use <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.tryfondo.com/">Fondo<a/> and they are unbelievably good. They basically act as an outsourced accounting team, and you submit everything through a shared Slack channel with them or via an online portal, which is remarkably straightforward to use. Fondo will handle your tax returns, annual filing and also send you monthly management reports. They charge on a sliding scale based on your expenses, so I’m not sharing what we pay here for confidentiality reasons.
+
+There is no need to do Xero or QuickBooks yourself - it is not a good use of your time, even if it is cheaper. The exception is if you have a finance background, in which case you can probably manage this yourself just fine (but even then, I’d question if it’s the best thing you could be doing).
+
+### Budgeting/Forecasting
+
+Don’t spend too much time making financial forecasts at an early stage - your priority should be to put together something credible to make sure you don’t run out of money in 3 months. Investors need you to have some idea of what's going on financially, but it's a complete waste of time to pretend you have a detailed view of what will happen.
+
+A relatively new tool called <a target="_blank" rel="noopener noreferrer nofollow" href="http://pry.co/">Pry</a> has been a game-changer for us. Yes, we all know you can “just” do it in Excel (which you will want to if you have a finance background), but a tool like Pry makes it so much easier. And you can skip the 2 years chained to your desk at Goldman!
+
+Pry connects seamlessly with your accounting software, and you then just fill out assumptions manually or using very simple formulae. Pry also has neat dashboards for visualising everything, as well as scenario modelling. They currently charge $100/month, though worth asking for a discount if you are a startup...
+
+## Where to put the money
+
+<iframe src="https://giphy.com/embed/3o6ZtpoNGDS4cZ8GWY" width="480" height="338" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/southparkgifs-3o6ZtpoNGDS4cZ8GWY">via GIPHY</a></p>
+
+Put it in <a target="_blank" rel="noopener noreferrer nofollow" href="https://www.brex.com/">Brex</a>, obviously (US only). They also make it incredibly easy to manage virtual credit cards, and their built-in expense management negates the need for something like Expensify. They don’t charge you anything.
```

Comment:
I'd wonder "but surely there's some way to get some interest" - perhaps mention @piemets 's findings that there isn't a good way to do this and why, with a one liner

Reply:
"I know what you're thinking - surely there's somewhere I can put my money to get some interest? The reality is that shopping around furiously to find some paltry 0.75% interest rate is highly unlikely to make any difference to your burn rate, and you're better off (in the early days) prioritising having your cash easily accessible vs. trying to optimise for returns. I am not a financial advisor though - you should talk to one if you decide this is important."

Reply:
"I know what you're thinking - surely there's somewhere I can put my money to get some interest? The reality is that TSLA stonks and you should put most of your runway into stocks. Let Technoking Elon take you to the Moon. 420 blaze it."